### ISSUE TITLE
Can't lookup email addresses

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#858]
https://octodon.social/@joeyh/99151251087460537

**Comment by Changaco** [comment]
I think we should introduce a new privacy setting "Allow anyone who knows my email address to find my profile" (which would be disabled by default) so that the email lookup tool could later be used for #456.

**Comment by Changaco** [comment]
However for now we just need a simple form consisting of a `<textarea>` where you can past a list of email addresses, and which shows you whether each address is currently associated to an Liberapay account or not.

**Comment by LiamDawe** [comment]
Yup, like this a lot, could end up being really useful. A very simple feature too. Would offer to help, but I'm limited to PHP.

**Comment by Juraj-Masiar** [comment]
I would also like to migrate my Patreon users here but if I can't identify individual users then how do I know who is who?
